app/test_main.py

```diff
@@ -1,3 +1,33 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_zero_if_zero_input() -> None:
```

Comment:
Consider using `@pytest.mark.parametrize` to parameterize this test and others. This will help reduce redundancy and make your tests cleaner and more maintainable as per the checklist requirements .